src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -440,38 +387,173 @@ impl<S> DdlServiceImpl<S>
 where
     S: MetaStore,
 {
-    async fn create_mview_on_compute_node(
+    fn get_dependent_relations(&self, fragment_graph: &StreamFragmentGraph) -> RwResult<Vec<u32>> {
```

Comment:
How about making this function static? Seems `self` is unused.
```suggestion
    fn get_dependent_relations(fragment_graph: &StreamFragmentGraph) -> RwResult<Vec<TableId>> {
```

Reply:
I agree, just fixed it 👍